inst/include/common/information.hpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * File:   information.hpp
+ *
+ * Author: Matthew Supernaw, Andrea Havron
+ * National Oceanic and Atmospheric Administration
+ * National Marine Fisheries Service
+ * Sustainable Fisheries Division
```

Comment:
I think we should leave this as just NMFS since we will have people from OST and many other science centers, and to my knowledge we're not the sustainable fisheries division anymore?